src/scenes/varrock.js

```diff
@@ -20,6 +20,12 @@ export class VarrockScene extends EnemyLevelScene {
                     path: "src/assets/sprites/DarkWizard.png",
                     maxHealth: 12,
                     killGold: 10
+                },
+                {
+                    name: "guard",
+                    path: "src/assets/sprites/VarrockGuard.png",
```

Comment:
I'm pretty sure this image is from future updates. Use the image you have for Falador guard for these. Much much much more nostalgic

Reply:
Also please rename falador guard to just guard

Reply:
That's what I was thinking but I opened up osrs and that's what they have rn. I can switch it though. Do you want to have the same guards for both Varrock and Falador?

Reply:
Imo it should be the same.

Reply:
Aye aye cap'n